<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qwSettings</class>
 <widget class="QWidget" name="qwSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>&amp;Free spins</string>
       </property>
       <property name="buddy">
        <cstring>qsbSameDozenColumnBeforeBet</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QSpinBox" name="qsbSameDozenColumnBeforeBet"/>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>on same dozen / column before bet</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Bet on</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QRadioButton" name="qrbBetOnDozenColumn">
         <property name="text">
          <string>Dozen / column</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="qrbBetOnRandom">
         <property name="text">
          <string>Random</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Progression</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="qrbProgressionAuto">
        <property name="text">
         <string>&amp;Auto</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="qsbProgression">
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="qrbProgressionManual">
        <property name="text">
         <string>&amp;Manual</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="qleProgression"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>F&amp;ree spins</string>
        </property>
        <property name="buddy">
         <cstring>qsbSameDozenColumnProgression</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QSpinBox" name="qsbSameDozenColumnProgression"/>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>on same dozen / column before progression</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="qcbProgressionDozenColumnNotChanged">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>&amp;Continue only when dozen / column not changed</string>
        </property>
        <property name="buddy">
         <cstring>qcbProgressionDozenColumnNotChanged</cstring>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>qsbProgression</zorder>
     <zorder>qrbProgressionManual</zorder>
     <zorder>qleProgression</zorder>
     <zorder>qrbProgressionAuto</zorder>
     <zorder>label_4</zorder>
     <zorder>qcbProgressionDozenColumnNotChanged</zorder>
     <zorder>label_5</zorder>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
